octopoes/octopoes/xtdb/query.py

```diff
@@ -231,8 +231,12 @@ def _where_field_in(self, ref: Ref, field_name: str, values: list[str]) -> None:
             value = value.replace('"', r"\"")
             new_values.append(f'"{value}"')
 
+        types_to_check = [ooi_type]
+        if ooi_type in get_abstract_types():
+            types_to_check = ooi_type.strict_subclasses()
```

Comment:
I would put this in an if-else clause so `types_to_check` won't be assigned twice if given OOI type is an abstract type